neuron_explainer/models/transformer.py

```diff
@@ -697,12 +698,10 @@ def load(
         simplify: bool = False,
         simplify_kwargs: dict[str, Any] | None = None,
     ) -> "Transformer":
-        if bf.exists(name_or_path):
+        if name_or_path.startswith("https://"):
             path = name_or_path
         else:
-            path = f"az://openaipublic/neuron-explainer/subject-models/{name_or_path.replace('-', '/')}"
-        if not bf.exists(path):
```

Comment:
I couldn't find a good way to check if any files exist in a directory like `https://openaipublic.blob.core.windows.net/neuron-explainer/subject-models/`. hitting it using `urllib.request` returns 404, there is no xml data associated with the directory, etc.